src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -232,7 +232,8 @@
     <xsd:complexType name="workflow">
         <xsd:sequence>
             <xsd:element name="marking-store" type="marking_store" />
-            <xsd:element name="support" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="support" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="support-strategy" type="xsd:string" minOccurs="0" maxOccurs="1" />
```

Comment:
but this is inconsistent with **all** configuration we have in Symfony. We use a child element only for complex values (array nodes in the Configuration class)